Source/WebKit/UIProcess/Network/NetworkProcessProxy.cpp

```diff
@@ -564,7 +564,7 @@ void NetworkProcessProxy::didFinishLaunching(ProcessLauncher* launcher, IPC::Con
         return;
     }
     
-#if PLATFORM(IOS_FAMILY)
+#if PLATFORM(COCOA) && USE(RUNNINGBOARD)
```

Comment:
All cases of "PLATFORM(COCOA) && USE(RUNNINGBOARD)" can change to just "USE(RUNNINGBOARD)". Runningboard only exists on Cocoa platforms.

Reply:
Got it. I thought it may be more readable this way for non-Apple platform developers unfamiliar with RunningBoard but it does seem pretty redundant now. I'll remove PLATFORM(COCOA)